### DISCUSSION TITLE
Using this for a wind sensor.

### DISCUSSION THREAD

**Discussion by dreeas** [Petrichor-Labs/nmea_data_convert#3] [Q&A]
Hi Thomas,

I have a project with a wind sensor that outputs nmea that I would like to post directly to a database.
Will your code work with nmea sentences for wind like the one below?
$IIMWV,305,R,000.74,M,A*15 

Br
Andreas

**Comment by nabelekt** [comment]
Hey, Andreas. Thanks for your question. My library relies on the [pynmea2](https://github.com/Knio/pynmea2) library to do the actual parsing of individual NMEA sentences. If the sentence type is not supported by pynmea2, it won't work. But if it is, I think it should work just fine! And good news! Looking briefly, I do see what looks like support for IIMWV sentences in pynmea2. I'd say give it a whirl and see what happens. Maybe try outputting to a CSV file first since that is a simpler operation. If you run into problems, please open an issue here.
